protocols/desktop-shell.xml

```diff
@@ -18,6 +18,7 @@
     </request>
 
     <request name="set_lock_surface">
+      <arg name="output" type="object" interface="wl_output"/>
```

Comment:
This is where we modify the protocol. The output is sent along with the surface, so it knows which one to bind to.